docs/howto/krustlet-on-lke.md

```diff
@@ -0,0 +1,281 @@
+LKE](kubernetes-on-lke.md) for more information.
+
+This specific tutorial will be running Krustlet on a Linode (VM); however
+you may follow these steps from any device that can start a web server on an IP
+accessible from the Kubernetes control plane.
+
+In the [how-to guide for running Kubernetes on LKE](kubernetes-on-lke.md),
+several environment variables were used. Let's reuse those values:
+
+```console
+$ CLUSTER_ID=[[CLUSTER-ID]]
+$ REGION="us-west"
+$ VERSION="1.18"
+$ TYPE="g6-standard-1"
+```
+
+Let's confirm that the cluster exists:
+
+```console
+$ linode-cli lke cluster-view ${CLUSTER_ID}
+```
+
+## Step 1: Create Linode (VM)
+
+In order to create a Linode, we will need to determine several values. These
+values may be determined with the following commands:
+
+```console
+$ linode-cli images list
+$ linode-cli regions list
+$ linode-cli linodes types
+```
+
+It is recommended that you use SSH keys to authenticate against the Linode. The
+[instructions](https://www.linode.com/docs/guides/use-public-key-authentication-with-ssh/) will guide you.
+
+Please set the value of `PUBLIC_KEY` to point to the file containing the public
+key you will use to authenticate the Linode:
+
+You will also need to identify the Linode user. You can do this with the following
+command. Set the value of `USER` to the user you wish to use:
+
+```console
+$ linode-cli show-users
+```
+
+You can create a Linode (VM) with the following command to create a Linode in the
+`us-west` region using a `g6-standard-1` type with a randomly-generated root
+password and your SSH public key:
+
+```console
+$ LABEL=[[YOUR-LABEL]]
+$ TYPE="g6-standard-1"
+$ REGION="us-west"
+$ IMAGE="linode/debian10"
+$ PASSWORD="$(</dev/urandom tr -dc a-z0-9 | head -c "${1:-32}";echo;)"
+
+$ linode-cli linodes create \
+  --type=${TYPE} \
+  --region=${REGION} \
+  --image=${IMAGE} \
+  --root_pass=${PASSWORD} \
+  --authorized_keys="$(cat ${PUBLIC_KEY})" \
+  --booted=true \
+  --label=${LABEL} \
+  --authorized_users=${USER}
+```
+
+You may then grab the Linode's `ID` and public IP `IP` using:
+
+```console
+$ LINODE_ID=$(linode-cli linodes list \
+  --label=${LABEL} \
+  --format=id \
+  --no-headers \
+  --text) && echo ${LINODE_ID}
+
+$ IP=$(linode-cli linodes list \
+  --label=${LABEL} \
+  --format=ipv4 \
+  --no-headers \
+  --text) && echo ${IP}
+```
+
+## Step 2: Get a bootstrap config for your Krustlet node
+
+Krustlet requires a bootstrap token and config the first time it runs. Follow
+the guide [here](bootstrapping.md), setting the `CONFIG_DIR` variable to `./`,
+to generate a bootstrap config and then return to this document. If you already
+have a kubeconfig available that you generated through another process, you can
+proceed to the next step. However, the credentials Krustlet uses must be part of
+the `system:nodes` group in order for things to function properly.
+
+NOTE: You may be wondering why you can't run this on the VM you just
```

Comment:
This is super minor, but it looks like there are 3 different kinds of NOTE blocks in this doc. This one, a block quote (`>`) version, and a bolded `**NOTE**`.  Could you standardize them all to the same style (probably the bold or block quote as it makes it stand out) for easier consumability?

Reply:
Apologies, cutting-and-pasting and not checking.

Fixed: https://github.com/deislabs/krustlet/pull/497/commits/e0269266607eaa4ac5c2af5f07dc76a85b2f34cf